elements/lisk-cryptography/src/convert.ts

```diff
@@ -23,6 +23,43 @@ import { hash } from './hash';
 // eslint-disable-next-line @typescript-eslint/no-require-imports
 import reverse = require('buffer-reverse');
 
+const CHARSET = 'zxvcpmbn3465o978uyrtkqew2adsjhfg';
+
+export const convertUIntArray = (
+	uintArray: number[],
+	fromBits: number,
+	toBits: number,
+): number[] => {
+	// eslint-disable-next-line no-bitwise
+	const maxValue = (1 << toBits) - 1;
+	let accumulator = 0;
+	let bits = 0;
+	const result = [];
+	// eslint-disable-next-line
+	for (let p = 0; p < uintArray.length; ++p) {
```

Comment:
```suggestion
	for (let p = 0; p < uintArray.length; p += 1) {
```
and remove the above `// eslint-disable-next-line`

Reply:
I can change to `p += 1` but there is another `lint` issue, so the disable-lint will remain regardless.

Reply:
actually you could change this if you want `for (const p of uintArray) {` as your doing the same in `polymod`

Reply:
There is a subtle difference between `for of` and the original code. If I change to `for of` the resulting bytes are not the same and the function will not work. That is the reason I am keeping it.